app/controllers/user_controller.py

```diff
@@ -1,18 +1,72 @@
+from http import HTTPStatus
+
+from app.configs.database import db
+from app.models.address_model import AddressModel
+from app.models.city_model import CityModel
+from app.models.user_model import UserModel
+from flask import jsonify, request
+from sqlalchemy.orm import Session
+from werkzeug.exceptions import NotFound
```

Comment:
Some of those imports doesn't look sorted, but in the other files they are all right, we might wanna take a look if `isort .` works different to any of us just to be sure.

Reply:
Done.